### DISCUSSION TITLE
Removing support for the Flow deck from the firmware breaks the LED ring deck

### DISCUSSION THREAD

**Discussion by mikendu** [bitcraze/discussions#70] [Q&A]
I was customizing the firmware using the new Kbuild system, and I removed support for a bunch of expansion decks I wasn't planning on using, but after I flashed the crazyflie with the firmware, I noticed I no longer had the ability to control the LED ring deck via parameters. I was unable to set the ring effect or control individual LEDs, or turn on the LED headlight anymore, and it was only after some trial and error that I discovered that the ability to do this appeared to depend on Flow deck support being included in the firmware.

For now, I've just included that, since I need the ability to set LEDs dynamically, but just thought I'd bring some attention to this issue.

**Comment by knmcguire** [comment]
Thanks for letting us know! Could you tell us a bit more how you removed the support? Did you remove it from the kbuild files so it will just not build or physically removed the code that runs the drivers for the decks. 

It will be a bit difficult for us to determine unless we know exactly what you removed.
